.github/workflows/release.yml

```diff
@@ -37,20 +37,21 @@ jobs:
       - name: create release
         if: ${{ steps.check_release.outputs.missing == 'true' }}
         id: create_release
-        uses: actions/create-release@v1.1.4
+        uses: ncipollo/release-action@v1.14.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          tag_name: "${{ steps.read-version.outputs.tag }}"
-          release_name: Changelog Enforcer ${{ steps.read-version.outputs.version }}
+          tag: "${{ steps.read-version.outputs.tag }}"
+          name: Changelog Enforcer ${{ steps.read-version.outputs.version }}
           body: ${{ steps.changelog_reader.outputs.changes }} 
           draft: false
           prerelease: false
+          token: ${{ env.GITHUB_TOKEN }}
```

Comment:
Do we need to then remove the `GITHUB_TOKEN` in the `env` section?

Reply:
If we want to keep it this way, then no, we need the `env` section, but there are other alternatives, I just wanted to stay close to what there already was. Theoretically we don't even need to handle the token stuff, if the normal Github Token is used and it's not just a secret you created.
Alternative 1 (directly access the `GITHUB_TOKEN`):
```
      - name: create release
        if: ${{ steps.check_release.outputs.missing == 'true' }}
        id: create_release
        uses: ncipollo/release-action@v1.14.0
        with:
          tag: "${{ steps.read-version.outputs.tag }}"
          name: Changelog Enforcer ${{ steps.read-version.outputs.version }}
          body: ${{ steps.changelog_reader.outputs.changes }} 
          draft: false
          prerelease: false
          token: ${{ secrets.GITHUB_TOKEN }}
```
Alternative 2 (use `github.token` variable, would be redundant because that is the default `ncipollo/release-action` uses when nothing is specified):
```
      - name: create release
        if: ${{ steps.check_release.outputs.missing == 'true' }}
        id: create_release
        uses: ncipollo/release-action@v1.14.0
        with:
          tag: "${{ steps.read-version.outputs.tag }}"
          name: Changelog Enforcer ${{ steps.read-version.outputs.version }}
          body: ${{ steps.changelog_reader.outputs.changes }} 
          draft: false
          prerelease: false
          token: ${{ github.token }}
```
Alternative 3 (don't specify token at all making `ncipollo/release-action` use the default of `github.token`):
```
      - name: create release
        if: ${{ steps.check_release.outputs.missing == 'true' }}
        id: create_release
        uses: ncipollo/release-action@v1.14.0
        with:
          tag: "${{ steps.read-version.outputs.tag }}"
          name: Changelog Enforcer ${{ steps.read-version.outputs.version }}
          body: ${{ steps.changelog_reader.outputs.changes }} 
          draft: false
          prerelease: false
```

Reply:
I'm all about simplicity. If we can use the Alternative three and the default GitHub Token work, I'm all for it!

Reply:
It should work, if it doesn't I think you'd need to adjust it here: https://github.com/dangoslen/changelog-enforcer/settings/actions
![grafik](https://github.com/dangoslen/changelog-enforcer/assets/30803034/d6bd43ca-9428-47bd-ad46-3fc140fe33ac)